### PR TITLE
OCPBUGS#11575: Document Operator SDK installation for macOS

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.12+

Issue: https://issues.redhat.com/browse/OCPBUGS-11575

Link to docs preview: https://58866--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/osdk/cli-osdk-install.html

https://58866--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/osdk/cli-osdk-install.html

https://58866--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/osdk-installing-cli.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by KeenonLee — COMMENTED** [review]
/lgtm

**Comment by xenolinux** [comment]
@KeenonLee Is this change valid for 4.9+ ocp versions?

**Comment by KeenonLee** [comment]
We can support darwin/arm64 client from 4.12.

File: cli_reference/osdk/cli-osdk-install.adoc [hunk 1]
```diff
@@ -8,7 +8,7 @@ toc::[]
 
 The Operator SDK provides a command-line interface (CLI) tool that Operator developers can use to build, test, and deploy an Operator. You can install the Operator SDK CLI on your workstation so that you are prepared to start authoring your own Operators.
 
-Operator authors with cluster administrator access to a Kubernetes-based cluster, such as {product-title}, can use the Operator SDK CLI to develop their own Operators based on Go, Ansible, or Helm. link:https://kubebuilder.io/[Kubebuilder] is embedded into the Operator SDK as the scaffolding solution for Go-based Operators, which means existing Kubebuilder projects can be used as is with the Operator SDK and continue to work.
+Operator authors with cluster administrator access to a Kubernetes-based cluster, such as {product-title}, can use the Operator SDK CLI to develop their own Operators based on Go, Ansible, Java, or Helm. link:https://kubebuilder.io/[Kubebuilder] is embedded into the Operator SDK as the scaffolding solution for Go-based Operators, which means existing Kubebuilder projects can be used as is with the Operator SDK and continue to work.
```
Comment (jldohmann) [inline cli_reference/osdk/cli-osdk-install.adoc:11]: I don't see this addition of Java reflected in the preview:
Screenshot 2023-04-19 at 10 06 16 AM
am I missing something?

**Review by jldohmann — COMMENTED** [review]
LGTM, though I did leave a question below...feel free to ping me again for merge, thank you!

File: cli_reference/osdk/cli-osdk-install.adoc (hunk 1 above)
Comment (xenolinux) [reply cli_reference/osdk/cli-osdk-install.adoc:11]: https://58866--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/osdk/cli-osdk-install.html ==> It's now showing here.

Comment (xenolinux) [reply cli_reference/osdk/cli-osdk-install.adoc:11]: Updated comment 0 with the preview link.

File: modules/osdk-installing-cli-macos.adoc [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+ifdef::openshift-origin[]
+* link:https://docs.docker.com/install/[`docker`] v17.03+, link:https://github.com/containers/libpod/blob/master/install.md[`podman`] v1.2.0+, or link:https://github.com/containers/buildah/blob/master/install.md[`buildah`] v1.7+
+endif::[]
+ifndef::openshift-origin[]
+* `docker` v17.03+, `podman` v1.9.3+, or `buildah` v1.7+
+endif::[]
+
+.Procedure
+
+. For `amd64` and `arm64` architectures, navigate to the link:https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/operator-sdk/[OpenShift mirror site for `amd64` architecture] and link:https://mirror.openshift.com/pub/openshift-v4/arm64/clients/operator-sdk/[OpenShift mirror site for `arm64` architecture] respectively.
```
Comment (mburke5678) [inline modules/osdk-installing-cli-macos.adoc:24]: Should this step refer to `x86_64` and `arm64` in order to match the two links: 

> For `x86_64` and `arm64` architectures, navigate to the link:https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/operator-sdk/[OpenShift mirror site for `x86_64` architecture] and link:https://mirror.openshift.com/pub/openshift-v4/arm64/clients/operator-sdk/[OpenShift mirror site for `arm64` architecture] respectively.

**Comment by mburke5678** [comment]
@xenolinux Can you address my one pesky nit? Otherwise looks good to merge.

**Pull request merged**